src/components/Dropdown/DropdownInput.tsx

```diff
@@ -80,7 +86,7 @@ export function DropdownInput(props: {
       id={screenReaderUUID && generateDropdownId(screenReaderUUID, -1)}
       autoComplete='off'
       aria-describedby={screenReaderUUID}
-      aria-activedescendant={screenReaderUUID && generateDropdownId(screenReaderUUID, focusedIndex)}
+      aria-activedescendant={!isTyping ? (screenReaderUUID && generateDropdownId(screenReaderUUID, focusedIndex)) : ''}
```

Comment:
can this just be `!isTyping && screenReaderUUID && generateDropdownId(screenReaderUUID, focusedIndex)`?

Reply:
when i try this I get an error: 

Types of property ''aria-activedescendant'' are incompatible.
      Type 'string | false | undefined' is not assignable to type 'string | undefined'.
        Type 'boolean' is not assignable to type 'string'.

Reply:
oh can you try switching the order of `screenReaderUUID` and `!isTyping`?

Reply:
this: `screenReaderUUID && !isTyping && generateDropdownId(screenReaderUUID, focusedIndex)`? it still doesn't work :c

Reply:
ok nvm then, the ternary operator makes it look a little uglier, but it's fine

Reply:
the && operator returns the first falsy value, so if screenReaderUUID is falsy it will return blank string since screenReaderUUID is a string, but if screenReaderUUID is truthy but !isTyping is falsy it will return false since isTyping is a boolean.

Maybe we can tweak `generateDropdownId` to accept an undefined screenReaderUUID, and return blank string if it's undefined? Normally it's good to keep utility methods "dumb" but this is an internal only method, and we do this `screenReaderUUID && generateDropdownId` check literally every time we call `generateDropdownId` so it's fine to bake in.
then for aria-activedescendant we can do
```ts
!isTyping ? generateDropdownId(screenReaderUUID, focusedIndex) : ''
// or
!isTyping && generateDropdownId(screenReaderUUID, focusedIndex) || ''
```